sharding/indexHashedNodesCoordinator.go

```diff
@@ -11,6 +11,7 @@ import (
 )
 
 type indexHashedNodesCoordinator struct {
+	doExpandEligibleList    func(uint32) []Validator
```

Comment:
this is not needed.

Reply:
it is needed to be able to change the "expandEligibleList" behaviour in the nodesCoordinatorWithRater